pandas/core/arrays/period.py

```diff
@@ -539,10 +541,13 @@ def _sub_period(self, other):
 
         return new_data
 
+    Other_types = Union[ABCPeriodArray, ABCSeries, ABCPeriodIndex,
```

Comment:
Are you just creating this alias to get the type on one line? Think you can use implicit line breaking instead